.releaserc

```diff
@@ -0,0 +1,38 @@
+{
+    "branch": "master",
```

Comment:
Everything looks good, doesn't this require the NPM_TOKEN and the GIT credentials stored as environment variables or something ?

Reply:
Thanks @MadaraUchiha-314 I saw that npm token and github token were already being used as part of the publish step in circle ci config. So I assumed that the tokens were already set and configured for your repo in circleCI. Please let me know if that is not the case and we can work together to configure them.

Reply:
@MadaraUchiha-314 Can you please help me understand if any additional changes are needed?

Reply:
Hi @MadaraUchiha-314.. Any update on this? If the valid tokens are already setup for your repo in circleCI, I think this can be merged. What do you think?

Reply:
![Screenshot 2020-10-17 at 2 19 11 PM](https://user-images.githubusercontent.com/6977429/96332794-c530c500-1083-11eb-90ad-138a2c27fba0.png)

The credential are currently stored as the following environment variables.

Does semantic release also use the same environment variable names is the only question I have

Reply:
@MadaraUchiha-314 Yes..that should be alright. You have the NPM token set for publishing the package, and a GitHub token that has rights to the repo. As per the instructions from https://itnext.io/automate-your-releases-versioning-and-release-notes-with-semantic-release-d5575b73d986 and https://semantic-release.gitbook.io/semantic-release/usage/ci-configuration, semantic versioning should work fine on merge. 
 Incase if there are any issues, they would most likely will only be with invalid environment variables on circleCI, and we can easily resolve them by creating new environment variables in circleCI settings if needed. Can you please merge this PR so that we can verify?

Reply:
@skodamarthi I looked at the documentation for semantic release and I have added the variable `GH_TOKEN` to the circle CI environment variables.